examples/validating-public-input/sp1/fibonacci/lib/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+use alloy_sol_types::sol;
+
+sol! {
+    /// The public values encoded as a struct that can be easily deserialized inside Solidity.
+    struct PublicValuesStruct {
+        uint32 n;
+        uint32 a;
+        uint32 b;
+    }
+}
+
+/// Compute the n'th fibonacci number (wrapping around on overflows), using normal Rust code.
+pub fn fibonacci(n: u32) -> (u32, u32) {
```

Comment:
I think we can remove this lib. From my understanding these are not used in the example